src/module/helpers.ts

```diff
@@ -36,8 +36,12 @@ export const registerHelpers = async function () {
     return Math.round(parseFloat(weight) / 100) / 10;
   });
 
-  Handlebars.registerHelper("getTagIcon", function (tag) {
-    return CONFIG.OSE.tag_images[tag as InventoryItemTag];
+  Handlebars.registerHelper("getTagIcon", function (tagValue: string) {
```

Comment:
Changes to this helper are the "actual" fix for the bug.

The rest is to get all paths to `/system/<system_id>/assets` to display correctly during development